src/include/sof/audio/selector.h

```diff
@@ -45,6 +53,20 @@ struct micsel_data {
 	struct ipc4_base_module_cfg base_cfg;
 	struct ipc4_audio_format output_format;
 };
+
+enum ipc4_selector_config_id {
```

Comment:
Pls document in dox format to have a complete IPC4 interface documentation.

Reply:
Done.